pkg/localstorev2/internal/chunkstamp/chunkstamp.go

```diff
@@ -47,7 +46,7 @@ type item struct {
 
 // ID implements the storage.Item interface.
 func (i *item) ID() string {
-	return path.Join(string(i.stamp.BatchID()), string(i.stamp.Index()))
+	return path.Join(string(i.stamp.BatchID()), string(i.stamp.BatchID()))
```

Comment:
Was this changed by mistake? It should be Index.

Reply:
you may be confusing this with the stampindex?